core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -101,6 +101,41 @@ angular.module('oppia').component('contributorDashboardPage', {
         ctrl.userCanReviewTranslationSuggestionsInLanguages = [];
         ctrl.userCanReviewVoiceoverSuggestionsInLanguages = [];
         ctrl.userCanReviewQuestions = false;
+        ctrl.defaultHeaderVisible = true;
+        ctrl.collapsibleHeaderVisible = false;
+        ctrl.scrollTest = false;
+
+        var $win = angular.element($window);
+        var defaultHeader = angular.element(document.querySelector(
+          '#default-header'));
+        var collapsibleHeader = angular.element(document.querySelector(
+          '#collapsible-header'));
+
+        window.onscroll = function() {
+          ctrl.scrollFunction();
+        };
+
+        ctrl.scrollFunction = function() {
+          if ($win.scrollTop() >= 5) {
+            defaultHeader.removeClass('oppia-fade-in');
+            defaultHeader.addClass('oppia-contributor-dashboard-header');
+            collapsibleHeader.removeClass(
+              'oppia-contributor-dashboard-collapsible-header');
+            collapsibleHeader.addClass(
+              'oppia-fade-in');
```

Comment:
Instead of directly adding/removing CSS classes like this, use AngularJS's built in [ng-class](https://docs.angularjs.org/api/ng/directive/ngClass) to conditionally add classes based on conditional expressions.

In this case, you can set `defaultHeaderVisible` when the scrollTop changes and then add the classes you need in the html based on the value of `defaultHeaderVisible`. Also, since only one type of header should be visible at one time, you don't need both `defaultHeaderVisible` and `collapsibleHeaderVisible`.

Reply:
Done